src/volue/mesh/_common.py

```diff
@@ -479,6 +479,24 @@ def _to_proto_attribute_masks(
     return attributes_masks
 
 
+def _to_proto_attribute_field_mask(
```

Comment:
docstring explaining that empty field mask returns everything and non full attribute info returns everything except definition?